eng/Build.proj

```diff
@@ -35,7 +35,7 @@
 
     <ItemGroup>
       <BuildProject Include="$(RepoRoot)src/**/*.csproj;$(RepoRoot)src/**/*.vbproj" Condition="'$(SrcProjects)' == 'true'" />
-      <BuildProject Include="$(RepoRoot)tests/**/*.csproj;$(RepoRoot)src/**/*.vbproj" Condition="'$(TestProjects)' == 'true'" />
+      <BuildProject Include="$(RepoRoot)tests/**/*.csproj;$(RepoRoot)tests/**/*.vbproj" Condition="'$(TestProjects)' == 'true'" />
```

Comment:
This is a bugfix that is in Tom's `dotnet test` PR, but that PR hasn't been merged yet.